Core/Object Arts/Dolphin/MVP/Views/Common Controls/OS.CCITEM.cls

```diff
@@ -87,7 +87,7 @@ mask: anInteger
 	bytes dwordAtOffset: _OffsetOf_mask put: anInteger!
 
 maskIn: anInteger
-	"OR in to contents' mask member the bit flags represented by anInteger"
+	"OR in to content's mask member the bit flags represented by anInteger"
```

Comment:
Same treatment for this and all the other cases below of "content's blah member", translate to "the blah field"